package.json

```diff
@@ -1,34 +1,43 @@
 {
-	"name": "@rbxts/replicaservice",
-	"version": "1.0.0-ts.7",
-	"description": "ReplicaService types",
-	"main": "src/init.lua",
-	"homepage": "https://github.com/Mixu78/replicaservice-typings#readme",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/Mixu78/replicaservice-typings"
-	},
-	"scripts": {},
-	"keywords": [],
-	"author": "",
-	"license": "ISC",
-	"types": "src/index.d.ts",
-	"files": [
-		"src"
-	],
-	"publishConfig": {
-		"access": "public"
-	},
-	"devDependencies": {
-		"@rbxts/compiler-types": "^2.0.4-types.0",
-		"@rbxts/types": "^1.0.642",
-		"@typescript-eslint/eslint-plugin": "^5.28.0",
-		"@typescript-eslint/parser": "^5.28.0",
-		"eslint": "^8.17.0",
-		"eslint-config-prettier": "^8.5.0",
-		"eslint-plugin-prettier": "^4.0.0",
-		"eslint-plugin-roblox-ts": "^0.0.32",
-		"prettier": "^2.7.0",
-		"typescript": "^4.8.3"
-	}
+  "name": "@rbxts/replicaservice",
+  "version": "1.0.0-ts.8",
+  "description": "(Types) Selective state replication system made by MadStudio.",
+  "main": "src/init.lua",
+  "homepage": "https://github.com/Mixu78/replicaservice-typings#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Mixu78/replicaservice-typings"
+  },
+  "scripts": {},
+  "keywords": [
+    "replication",
+    "states",
+    "types",
+    "roblox"
+  ],
+  "author": "Mixu_78",
+  "contributors": [
+    "Sandy Stone"
+  ],
+  "license": "Apache-2.0",
+  "types": "src/index.d.ts",
+  "files": [
+    "src"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@rbxts/compiler-types": "^2.0.4-types.0",
+    "@rbxts/types": "^1.0.646",
+    "@typescript-eslint/eslint-plugin": "^5.42.0",
+    "@typescript-eslint/parser": "^5.42.0",
+    "eslint": "^8.26.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-roblox-ts": "^0.0.34",
+    "prettier": "^2.7.1",
+    "roblox-ts": "^2.0.4",
+    "typescript": "^4.8.4"
+  }
```

Comment:
What happened to the formatting here? You also seem to have changed the license and description, I think that's unnecessary.

Reply:
I've changed the `license` field to match up with the license of the repository (which it's Apache-2.0). The description was changed to be more "descriptive" about the module (ReplicaService) along with the name of the official creator to avoid confusion from other different packages with the same keyword "replica".

Reply:
In that case changing the license field is fine. In my opinion the description should simply state that the package contains types for ReplicaService, not go into details about what ReplicaService is.